engines/sword25/gfx/graphicengine.cpp

```diff
@@ -453,4 +456,8 @@ bool GraphicEngine::unpersist(InputPersistenceBlock &reader) {
 	return reader.isGood();
 }
 
+bool GraphicEngine::isRTL(){
```

Comment:
This doesn't follow our code formatting guidelines. Write `bool GraphicEngine::isRTL() {`

Reply:
Done